src/SFA.DAS.ReleaseTool.Infrastructure/Client/ReleaseClient.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Clients;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Contracts;
+using Microsoft.VisualStudio.Services.WebApi;
+using SFA.DAS.ReleaseTool.Core.Configuration;
+using SFA.DAS.ReleaseTool.Infrastructure.IReleases;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.ReleaseTool.Infrastructure.Releases
+{
+    public class ReleaseClient : IReleaseClient
+    {
+        private readonly VssConnection _vssConnection;
+        private readonly IOptions<VstsConfiguration> _configuration;
+
+        public ReleaseClient(IOptions<VstsConfiguration> configuration)
+        {
+            _configuration = configuration;
+
+            _vssConnection = SetConnection();
+        }
+
+        private VssConnection SetConnection()
+        {
+            VssCredentials creds = new VssBasicCredential(string.Empty, _configuration.Value.PatToken);
+
+            VssConnection connection = new VssConnection(new Uri(_configuration.Value.CollectionUri), creds);
+
+            return connection;
+        }
+
+        public List<ReleaseDefinition> GetReleases()
+        {
+            var client = _vssConnection.GetClient<ReleaseHttpClient>();
+
+            var release = client.GetReleaseDefinitionsAsync(_configuration.Value.ProjectName, expand: ReleaseDefinitionExpands.Variables).Result;
+
+            return release;
+        }
+
+        public ReleaseDefinition GetRelease(string releaseName)
+        {
+            var client = _vssConnection.GetClient<ReleaseHttpClient>();
+
+            var release = client.GetReleaseDefinitionsAsync(_configuration.Value.ProjectName, releaseName, expand: ReleaseDefinitionExpands.Variables).Result;
+
+            return release.FirstOrDefault();
+        }
+
+        public DeploymentStatus CheckReleaseStatus(int releaseDefinitionId, int releaseId)
+        {
+            ReleaseHttpClient releaseClient = _vssConnection.GetClient<ReleaseHttpClient>();
+
+            List<Deployment> deployments = releaseClient.GetDeploymentsAsync(project: _configuration.Value.ProjectName, definitionId: releaseDefinitionId, top: 5).Result;
+
+            var deployment = deployments.Single(r => r.Id == releaseId);
+
+            return deployment.DeploymentStatus;
+        }
+
+        public Release CreateRelease(int releaseDefinitionId, string ipAddress)
```

Comment:
IpAddress parameter isn't going to work if you intend to support other releases in future